### ISSUE TITLE
Merge all repositories

### ISSUE THREAD

**Issue by RDIL** [opensourcedesign/opensourcedesign.github.io#236]
As of #235 , it was learned that all the repositories that are not here do not have synced includes/assets/fonts etc.

## Status

- [x] Events
- [ ] Talks
- [ ] Jobs

@opensourcedesign/core any thoughts?

**Comment by Jc-L** [comment]
Should we also consider combine the /opensourcedesign.github.io into a properly named one (ie. /opensourcedesign.net) ? This was apparently to do as well.

**Comment by RDIL** [comment]
If we do that, then the subrepos wont work

**Comment by jancborchardt** [comment]
@Jc-L yes, we did rename the repo previously but it resulted in issues with the subsites indeed. I’m not sure about the reasons though, but we reverted to the previous names.

@RDIL but if the repos of events and jobs are inside this repository too, then the rename of this repo to `opensourcedesign.net` should work?

We should think about if there’s anything we need to be aware of except the repositories. Currently I can only think about these:
- We need to make sure that the job submission form (linked on the jobs page) uses the new repository to open the pull requests https://opensourcedesign.net/jobs/job-form/
- Need to make sure that the bot which automatically posts jobs to the "Job Postings" category in the forum still works https://discourse.opensourcedesign.net/c/job-postings
- The open pull requests in the jobs repository should be resolved, and the existing issues moved https://github.com/opensourcedesign/jobs/pulls
- Anything special needed for https://github.com/opensourcedesign/events/ ? We don’t have any linking as far as I know.

cc @opensourcedesign/website  anythign else?

**Comment by RDIL** [comment]
Yes, in that case, the rename could work.
@jancborchardt can you head to organization settings and enable issue transferring so we can begin this?

**Comment by jancborchardt** [comment]
Moved the 2 relevant issues from jobs into here, tagged with "Jobs": https://github.com/opensourcedesign/opensourcedesign.github.io/labels/Jobs

Reviewed 2 pull requests from jobs, only one left to review but is basically done: https://github.com/opensourcedesign/jobs/pull/385

cc @ei8fdb @m1guelpf @AnXh3L0 @dmichl @evalica @simonv3 about combining the repositories just FYI. :)

**Comment by jancborchardt** [comment]
Also I’d say let’s do it separately – first events as it’s less critical. And if that works all good, we can move jobs into here too. Sound good?

**Comment by RDIL** [comment]
Works for me - ill get events rolling

**Comment by RDIL** [comment]
#239 is now open - feel free to modify my branch

**Comment by SaptakS** [comment]
Based on discussions in monthly calls, following are the CTA:
- We probably still need to merge the talks repo into the main. Maybe we can merge the talks repos and presentation folder.
- Merge the jobs repo into this project.

**Comment by RDIL** [comment]
@SaptakS I can start on talks once #305 is merged (which can happen with one more approval, which can be yours).

**Comment by SaptakS** [comment]
Reviewed and merged #305

**Comment by jdittrich** [comment]
I personally would keep Jobs and Decks separate (to make it easier to do changes to them, particulary Website and Jobs). Thus I suggest to close this.

**Comment by jdittrich** [comment]
Closed upon "Close decision by end of 2025"-Tag
